### DISCUSSION TITLE
Quarto Document Language for Spanish NOT working.

### DISCUSSION THREAD

**Discussion by JuanRodCif** [quarto-dev/quarto-cli#1979] [Q&A]
Hello, I have used the lang option within the title block in a '.qmd' document, specifying that the language used is Spanish (es). Nevertheless, when I Render the file to either an HTML or a Docx format output file, the document does not incorporate the Spanish language characters. Could someone give me a hand? 

I am attaching the MS-Word output document, as well as the QMD file printed as PDF.
[QMD file printed as PDF.pdf](https://github.com/quarto-dev/quarto-cli/files/9378166/QMD.file.printed.as.PDF.pdf)

[CursoUniandesJR.docx](https://github.com/quarto-dev/quarto-cli/files/9378134/CursoUniandesJR.docx)

**Comment by cscheid** [comment]
I can't reproduce the issue on macOS. This is my output:

<img width="594" alt="image" src="https://user-images.githubusercontent.com/285675/185510984-90fe4f06-68e8-4888-93f8-713ee0b80558.png">

Can you file a github issue and fill out all of the information there so that we know your environment better?

**Reply by JuanRodCif** [reply]
Hi Carlos, thank you very much for your response. Just so that you know I am working with a Windows 11 PC. I will file a Github issue as you suggest.

**Reply by JuanRodCif** [reply]
Hello Carlos, per your request I have just filed the Github issue (#1999).
What I could not do, nevertheless, was to run the 'quarto check' and 'quarto tools check' as I don't know how to run those.
